packages/utils/paginate.js

```diff
@@ -118,3 +118,30 @@ module.exports.paginator = (args, payloadFn, nodesFn) => {
     _nodes: nodes,
   }
 }
+
```

Comment:
maybe it would be nice to have a short explanatory comment why we have magic pageini in addition to paginate aka pageini does that, paginator does this